.github/workflows/ci.yml

```diff
@@ -133,6 +133,7 @@ jobs:
           }
           
           echo "::set-output name=version::$main_version"
+          echo "::set-output name=tag::$tag"
```

Comment:
`version` and `tag` will have the same value here, so we should only output one of them (I'd argue for `version`).

Reply:
nice, on the closer inspection I can see what you mean ✅